fastmri/data/subsample.py

```diff
@@ -48,7 +48,13 @@ class MaskFunc:
     and ``EquispacedMaskFunc``.
     """
 
-    def __init__(self, center_fractions: Sequence[float], accelerations: Sequence[int]):
+    def __init__(
+        self,
+        center_fractions: Sequence[float],
+        accelerations: Sequence[int],
+        sync_center_fractions_accelerations: bool = True,
```

Comment:
The name could be improved (but I have no better name to propose except adding a `and` like this `sync_center_fractions_and_accelerations`).

Maybe by changing the default value to `False`, you might get a better name like `allow_any_combination` or something like this.